### PR TITLE
App crashes after sync #19655

### LINKED ISSUE #19655: App crashes after sync
### Steps:
1. `User A` generates a sync QR code
2. `User B` scans the QR of `User A`

### Actual result:
App is crashed

### Expected result:
Accounts are synced without crash

### OS: 
IOS, Android

### Devices:
- Pixel 7a, Android 13
- iPhone 11 Pro Max, IOS 17

### Logs
[logs (1).txt](https://github.com/status-im/status-mobile/files/14992728/logs.1.txt)

### PR DESCRIPTION
**Pull request by flexsurfer**
bug was introduced in this PR : https://github.com/status-im/status-mobile/pull/19467

fixes #19655

### CODE REVIEW

**Comment by qfrank** [comment]
Hi @flexsurfer , there were already a relate [PR](https://github.com/status-im/status-go/pull/5080) might fix this issue

**Review by qfrank — APPROVED** [review]
LGTM

File: src/utils/ethereum/chain.cljs [hunk 1]
```diff
@@ -49,33 +49,15 @@
...
   [network]
   (-> network
       network->chain-keyword
       name))
 
 (defn get-current-network
-  [m]
-  (get (:networks/networks m) (:networks/current-network m)))
+  [_]
+  nil)
```
Comment (Samyoul) [inline src/utils/ethereum/chain.cljs:60]: I'm a total noob, so humour my question, why is this func an no-op?

**Review by Samyoul — APPROVED** [review]

**Comment by pavloburykh** [comment]
@flexsurfer thanks a lot for the fix! PR is ready for merge.

**Pull request merged**